src/redisjson.rs

```diff
@@ -62,10 +63,16 @@ impl Path {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct ValueIndex {
+    pub key: String,
+    pub index: String,
+}
```

Comment:
I suggest naming this field `index_name` and not `index`; there's too many things `index` can refer to already.

Reply:
:+1: